checks.d/mongo.py

```diff
@@ -252,17 +258,35 @@ def check(self, instance):
             raise Exception(status['errmsg'].__str__())
 
         status['stats'] = db.command('dbstats')
+        dbstats = {}
+        dbstats[db_name] = {'stats': status['stats']}
+
+        tags = list(set(tags))
```

Comment:
Yeah we need to do it again, in fact we need to do it for every db in the mongo cluster, stats reported are specific to the db you call them on.